projects/client/src/services/cloudinary.service.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const uploadImage = async (imageFormData: FormData): Promise<any> => {
+  try {
+    const response = await axios.post('https://api.cloudinary.com/v1_1/dtywqmn9a/image/upload', imageFormData, {
```

Comment:
hhmmm... Please create a config module to hold your configuration, I mean... inside the src folder create a new folder called **config** with the following structure

```
export defaultConfig {
  apiUrl: 'https://api.cloudinary.com/v1_1/dtywqmn9a/image/',
  apiKey: 'API_KEY',
  apiSecret: 'API_SECRET',
  ...
};

```
Just an example